packages/react-core/src/components/NotificationBadge/NotificationBadge.tsx

```diff
@@ -6,6 +6,10 @@ import styles from '@patternfly/react-styles/css/components/NotificationBadge/no
 export interface NotificationBadgeProps extends ButtonProps {
   /**  Adds styling to the notification badge to indicate it has been read */
   isRead?: boolean;
+  /** Adds styling to the notification badge to indicate it needs attention */
+  isNeedingAttention?: boolean;
+  /** A number displayed in the badge alongside the icon */
+  count?: number;
```

Comment:
Can we just make this `number | string` and remove the `count > 0` condition? Someone might want to put something else in there.

Reply:
@mcarrano do we want to allow a sting here?

Reply:
Is there no use case for a string? e.g. `${count} messages`